draft-ietf-quic-transport.md

```diff
@@ -906,6 +906,11 @@ explained in more detail as they are referenced later in the document.
 | 0x10 - 0x17 | STREAM            | {{frame-stream}}            |
 {: #frame-types title="Frame Types"}
 
+All QUIC frames are idempotent.  That is, the state of the connection is
+unchanged regardless of whether a frame is processed one time or multiple times,
```

Comment:
How about: A valid frame does not change the state of the connection as a result of being received more than once.

Reply:
It could affect the state in terms of congestion logic and timeouts.

Reply:
@mikkelfj, presumably you are talking about receiving ACK frames. It shouldn't affect the state of congestion control or timeouts, since it all relies on whether an ACK frame has new information or not.

Reply:
I think that it's packets and (specifically) ACKs that affect congestion and loss recovery state.

Reply:
A valid frame does not cause undesirable side effects or errors when received more than once.

Reply:
I like this last re"framing"

Reply:
Idle timeout?